apps/storybook/src/forms/Checkbox.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Checkbox, Flex, Text } from "@optiaxiom/react";
+
+const meta: Meta<typeof Checkbox> = {
+  component: Checkbox,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Checkbox>;
+
+export const Basic: Story = {
+  args: {
+    children: "Label",
+  },
+};
+export const HelperText: Story = {
+  args: {
+    children: <>Label</>,
```

Comment:
Cant we just pass it as string? Do we need the wrapping? 

```suggestion
    children: "Label",
```

Reply:
We can. But there are stories in the string format here. So, tried all the possible cases for this

Reply:
Gotcha.

Reply:
we should have a separate story clearly testing this if that's our intent - something like:

```
export const ReactNodeChildren: Story = {
```

otherwise a future dev won't know why this wasn't a string and might change it unintentionally